### ISSUE TITLE
Chinese is garbled in translate icon

### ISSUE THREAD

**Issue by carlmjohnson** [tailwindlabs/heroicons#107]
Inside the translate icon, the symbol on the left is presumably meant to be 文, a Chinese character meaning word/language/writing. However, the diagonal upper-left to lower-right stroke does not connect to the horizontal line above it. This violates the rules of Chinese calligraphy and makes the symbol look garbled. It's halfway between 文 and 夕 (evening), which just looks wrong.  The stroke also doesn't extend far enough down, but that's fine because it looks like it's being covered up by the A. To fix the problem, just extend the stroke up and to the left so it touches the horizontal.

**Comment by carlmjohnson** [comment]
I'm coming to this from Japanese, and it looks like there's a slight difference in how 文 is written in Japanese vs. Chinese. https://en.wiktionary.org/wiki/文 I still think it would look better with the stroke connected given that the rest of the stroke is curving down instead of curving up, but adding an up curve would also be calligraphically valid.

**Comment by sschoger** [comment]
Hey!

This is interesting! 

This distinction seems to be a common pattern on other popular icon libraries like [Google](https://fonts.google.com/icons?icon.query=translate), [Font Awesome](https://fontawesome.com/icons/language?s=solid), and [Phosphor icons](https://phosphoricons.com/) to name a few:
![image](https://user-images.githubusercontent.com/2652515/155758756-e3915a8b-322a-4cba-85ce-fbcca8dfd8d6.png)

I'm going to leave it for now so it's consistent with the other libraries but I am in the process of updating the entire set and _may_ reevaluate this icon when I revisit it 👍🏻

**Comment by carlmjohnson** [comment]
Since I opened this issue in 2020, I have noticed that 文 has become the unofficial symbol for “translate” in lots of apps and systems (including iOS which gets it “right”, see below) and many of them have the calligraphic inconsistency I complained about. My guess is that this is because these icons are being created by people who don’t read hanzi/kanji and are just copying other icons. My preference would be that if 文 is going to be the icon for translate, it should look “right” but I’m probably just going to have to grit my teeth for the foreseeable future. 


![FDD193AA-7D16-4648-9A97-8F8BFD2DFC2C](https://user-images.githubusercontent.com/222245/155769443-5c8df59c-dc06-4832-b20f-c2ee9ab5ad81.jpeg)
